cypress/commons/actions/generic/TableParameters.js

```diff
@@ -87,6 +87,14 @@ function getFullscreenButton(tableParameterElement) {
   return getGrid(tableParameterElement).find(GENERIC_SELECTORS.genericComponents.table.toolbar.fullscreenButton);
 }
 
+function getAddRowButtonInput(tableParameterElement) {
+  return tableParameterElement.find(GENERIC_SELECTORS.genericComponents.table.toolbar.addRowButton);
+}
+
+function getDeleteRowsButtonInput(tableParameterElement) {
+  return tableParameterElement.find(GENERIC_SELECTORS.genericComponents.table.toolbar.deleteRowsButton);
+}
```

Comment:
There's no need to add `Input` in the names of functions `getAddRowButtonInput` and `getDeleteRowsButtonInput`, we're only selecting the buttons here :slightly_smiling_face:

Reply:
DONE